CONTRIBUTING.md

```
@@ -83,13 +83,26 @@ To check coverage locally:
 ```bash
 pytest --cov=src/phonometry --cov-report=term-missing tests/
 ```
-A few tests need reference material that is not committed (its licence forbids
-redistribution) and skip when it is absent. The EBU programme-loudness
-full-chain oracle (`tests/broadcast/test_ebu_material_oracle.py`) needs the free
-[EBU loudness test set](https://tech.ebu.ch/publications/ebu_loudness_test_set):
-drop it under `plan/dsp-sources/ebu-loudness-test-set/` or point
-`EBU_LOUDNESS_TEST_SET` at your copy. CI does not fetch it; the committed
-per-block series in `tests/data/broadcast/` cover the same cases without audio.
+### 3b. Oracle data (committed vs local)
```

markdownlint-cli2 (0.23.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)